swift-sdk/Internal/ApiClient.swift

```diff
@@ -218,7 +218,7 @@ extension ApiClient: ApiClientProtocol {
     }
     
     // MARK: - Embedded Messaging
-    func getEmbeddedMessages() -> Pending<SendRequestValue, SendRequestError> {
+    func getEmbeddedMessages() -> Pending<[IterableEmbeddedMessage], SendRequestError> {
         let result = createRequestCreator().flatMap { $0.createGetEmbeddedMessagesRequest() }
```

Comment:
Why is there a difference in the SendRequestValue for in-app vs. embedded?

Reply:
For embedded messaging, it'll automatically serialize it to that format, where as `SendRequestValue` is a generic return value.